### PR TITLE
use api show posts for detail product

### PR DESCRIPTION
**Pull request by at-trangle**
## Description
use api show posts for detail product
## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

File: public/css/public/custom.css [hunk 1]
```diff
@@ -98,4 +98,23 @@
...
+  display: none;
+}
+#posts-list .item .quick-reply .btn_add_comment{
+  margin-right: 10px;
+}
+#posts-list .item .diff_time{
+  text-align: center;
+  font-style: italic;
+  color: #d2691e;
 }
```
Comment (at-mailuong) [inline public/css/public/custom.css]: end line

File: public/js/public/posts.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    data.forEach(posts => {
+        let url = posts.image_path;
+        let image = posts.user.user_info.avatar;
+        let name = posts.user.user_info.full_name;
+        let diffTime = posts.diff_time;
+        let content = posts.content;
+        let stars = '';
+        if (posts.type == 1) {
+            let rate = Math.round(posts.rating);
+            for(i=1; i<= 5; i++){
```
Comment (at-mailuong) [inline public/js/public/posts.js]: space nha

File: public/js/public/posts.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                if(i <= rate) {
```
Comment (at-mailuong) [inline public/js/public/posts.js]: cũng space

File: public/js/public/posts.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        header: {
+            'Accept': 'application/json',
+        },
+        success: function(response) {
+            generatePosts(response.result.data);
+        }    
+    });
+};
+getAjax('/api' + $url + '/posts');
+$(document).ready(function(){
```
Comment (at-mailuong) [inline public/js/public/posts.js]: tương tự space

**Comment by at-mailuong** [comment]
LGTM

File: public/js/public/posts.js [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+function generatePosts(data) {
+    html = '';
+    data.forEach(posts => {
+        let url = posts.image_path;
+        let image = posts.user.user_info.avatar;
+        let name = posts.user.user_info.full_name;
+        let diffTime = posts.diff_time;
+        let content = posts.content;
+        let stars = '';
+        if (posts.type == 1) {
```
Comment (at-quangtran) [inline public/js/public/posts.js]: define 1

**Pull request merged**